packages/sign-plugin/src/utils/getCreatePluginVersion.ts

```diff
@@ -0,0 +1,21 @@
+import { existsSync, readFileSync } from 'node:fs';
+import { resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+const __dirname = fileURLToPath(new URL('.', import.meta.url));
+
+export const getCreatePluginVersion = () => {
+  try {
+    const crpcJSONPath = resolve(__dirname, '.config', '.crpc.json');
```

Comment:
Hmm... are you sure this resolves to the plugins `.cprc.json` file? 🤔

Reply:
Yes, but only if you run the `npx @grafana/sign-plugin@latest` inside the plugin directory.